core/services/telemetry/types.go

```diff
@@ -0,0 +1,8 @@
+package telemetry
+
+// FIXME: remove this
```

Comment:
does this still need to be removed?

Reply:
When the OCR stuff gets integrated, it can be removed.

Reply:
It's integrated now, is this comment still relevant?